### PR TITLE
[ci:docs] Update troubleshoot page

### PR DESCRIPTION
**Pull request by rhatdan**
Add information about podman with SELinux and using container_init_t.

Add informantion about Centos7 as well as RHEL7 init containers not working
on a cgroups V2 system.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Alternate to https://github.com/containers/libpod/pull/6195

File: troubleshooting.md [hunk 1]
```diff
@@ -424,9 +431,9 @@ Choose one of the following:
     * Install the fuse-overlayfs package for your Linux Distribution.
     * Add `mount_program = "/usr/bin/fuse-overlayfs"` under `[storage.options]` in your `~/.config/containers/storage.conf` file.
 
-### 17) rhel7-init based images don't work with cgroups v2
+### 17) RHEL 7 and CentOS 7 based `init` images don't work with cgroups v2
```
Comment (TomSweeneyRedHat) [inline troubleshooting.md:434]: I know this is old, but 'cgroups' -> 'cgroup' here and through out.

File: troubleshooting.md [hunk 2]
```diff
@@ -424,9 +431,9 @@ Choose one of the following:
...
 
-The systemd version shipped in rhel7-init doesn't have support for cgroups v2.  You'll need at least systemd 230.
+The systemd version shipped in RHEL 7 and CentOS 7 doesn't have support for cgroups v2.  Cgroup V2 support requires at least systemd 230.
```
Comment (TomSweeneyRedHat) [inline troubleshooting.md:436]: cgroup is supposed to be lower case everywhere.  How about the second sentence starts with "Further, cgroup v2".   Regardless, "V2" to "v2" for consistency.

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @QiWang19 @baude @mheon @giuseppe @vrothberg @jwhonce PTAL

File: troubleshooting.md [hunk 3]
```diff
@@ -424,9 +431,10 @@ Choose one of the following:
     * Install the fuse-overlayfs package for your Linux Distribution.
     * Add `mount_program = "/usr/bin/fuse-overlayfs"` under `[storage.options]` in your `~/.config/containers/storage.conf` file.
 
-### 17) rhel7-init based images don't work with cgroups v2
+### 17) RHEL 7 and CentOS 7 based `init` images don't work with cgroup v2
 
-The systemd version shipped in rhel7-init doesn't have support for cgroups v2.  You'll need at least systemd 230.
+The systemd version shipped in RHEL 7 and CentOS 7 doesn't have support for cgroup v2.  Support for cgroup V2 requires version 230 of systemd or newer, which
```
Comment (TomSweeneyRedHat) [inline troubleshooting.md:436]: once nit for consistency sake:
```suggestion
The systemd version shipped in RHEL 7 and CentOS 7 doesn't have support for cgroup v2.  Support for cgroup v2 requires version 230 of systemd or newer, which
```

File: troubleshooting.md [hunk 4]
```diff
@@ -219,8 +219,15 @@ the system.
 
 #### Solution
 
-SELinux provides a boolean `container_manage_cgroup`, which allows container
-processes to write to the cgroup file system. Turn on this boolean, on SELinux separated systems, to allow systemd to run properly in the container.
+Newer versions of Podman (2.0 or greater) support running init based containers
+with a different SELinux labels, which allow the container process access to the
+cgroup file system.  This feature requires container-selinux-2.132 or newer
```
Comment (QiWang19) [inline troubleshooting.md:224]: extra space before `This feature`

File: troubleshooting.md [hunk 5]
```diff
@@ -240,7 +247,7 @@ cannot find newuidmap: exec: "newuidmap": executable file not found in $PATH
 
 #### Solution
 
-Install a version of shadow-utils that includes these executables.  Note RHEL7 and Centos 7 will not have support for this until RHEL7.7 is released.
+Install a version of shadow-utils that includes these executables.  Note RHEL7 and CentOS 7 will not have support for this until RHEL7.7 is released.
```
Comment (QiWang19) [inline troubleshooting.md:250]: extra space ` Note RHEL7`

Comment (QiWang19) [reply troubleshooting.md:250]: there are `RHEL7` and `RHEL 7`, can we use the same format, with space before 7 or without space

Comment (rhatdan) [reply troubleshooting.md:250]: Yup

**Comment by QiWang19** [comment]
LGTM

**Pull request merged**